index.html

```diff
@@ -64,7 +53,7 @@ <h3>Append saved places here</h3>
         </nav>
     </div>
     
-    <script type="text/javascript" src="assets/javascript/main.js"></script>
+    <script type="text/javascript" src="assets/javascript/myfavorites.js"></script>
```

Comment:
KEEP THIS TOO: 
<script type="text/javascript" src="assets/javascript/main.js"></script>